be/src/storage/storage_engine.cpp

```diff
@@ -330,6 +332,50 @@ void StorageEngine::_start_disk_stat_monitor() {
     }
 }
 
+void StorageEngine::_start_report_profile() {
```

Comment:
It is not very suitable as a function of storage engine, report is for long-time/batch processing.